### ISSUE TITLE
Multiple Azure Information Protection tenant keys for O365 Administrative Units

### ISSUE THREAD

**Issue by ub44203** [MicrosoftDocs/Azure-RMSDocs#608]
Is it possible to use separate BYOK Azure tenant keys for separate O365 Administrative Units within the same O365 tenant?  Is it possible to limit access to the Administrative Unit BYOK access and administration from other Administrative Units within a single tenant?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: cf8412fc-3bea-8925-169c-24f2f85d5fb5
* Version Independent ID: 58c2c518-b291-70f0-c146-c116e8159c17
* Content: [Your Azure Information Protection tenant key](https://docs.microsoft.com/en-us/azure/information-protection/plan-implement-tenant-key)
* Content Source: [Azure-RMSDocs/plan-implement-tenant-key.md](https://github.com/Microsoft/Azure-RMSDocs/blob/master/Azure-RMSDocs/plan-implement-tenant-key.md)
* Service: **information-protection**
* GitHub Login: @cabailey
* Microsoft Alias: **cabailey**

**Comment by cabailey** [comment]
@ub44203 No, the Azure Information Protection tenant key secures all cryptographic functions for the tenant and you can only have one key active at once.

**Comment by cabailey** [comment]
#please-close
